templates/user/base-users-table.jade

```diff
@@ -14,22 +14,23 @@ thead
         block after_point_head
     tbody
         for rank, user in users
-            tr(id='user-{{ user.user.username }}')
-                td #{rank}
-                block after_rank
-                if show_organization
-                    td(class=user.display_rank)
-                        .userinfo
-                            - with org=user.organization
-                                if org
-                                    span.organization
-                                        a(href='{% url "organization_home" org.key %}')= org.short_name
-                            a.user-name(href='{% url "user_page" user.user.username %}')= user.user.username
-                            block user_data
-                else
-                    td.user-name(class=user.display_rank)
-                        a(href='{% url "user_page" user.user.username %}')= user.user.username
-                block before_point
-                block point
-                    td.user-points= user.points
-                block after_point
+            if not user.unlisted_user
```

Comment:
Instead of this, filter in the Python view code that generates `users`, since that's more efficient than a branch in a template.